xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/panels/LivedataAdvancedPanelFilterEntry.vue

```diff
@@ -114,7 +114,7 @@ export default {
   visibility: hidden;
   margin-left: 5px;
   padding: 6px 10px;
-  border-radius: 3px;
+  border-radius: @border-radius-small;
```

Comment:
Apparently here we did replace with a less variable so I don't understand why we didn't do it everywhere...